monkey/monkey_island/cc/services/zero_trust/report_data/finding_service.py

```diff
@@ -6,7 +6,7 @@
 from monkey_island.cc.services.zero_trust.monkey_findings.monkey_zt_details_service import MonkeyZTDetailsService
 
 
-class ZeroTrustFindingService:
+class FindingService:
```

Comment:
Because it's now used only in zero trust report resource and in file system is under `zero_trust_report` folder. I think this gives sufficient context

Reply:
I've referred to it in another comment. when a service is being accessed directly from a resource, it should have the full name, since the service is not part of the resource

Reply:
That resource being ZeroTrustReport. I think there's enough context to leave the shorter version.